mod/ckeditor/views/default/elgg/ckeditor.js

```diff
@@ -19,32 +19,43 @@ define(function (require) {
 	var config = require('elgg/ckeditor/config');
 
 	var elggCKEditor = {
-		/**
-		 * A flag that indicates whether handlers were registered
-		 */
-		ready: false,
-
```

Comment:
This merge is blocking my MD work so I'm merging. If this `ready: false` needs to go back in please make a ticket.